app/App.tsx

```diff
@@ -79,14 +79,16 @@ export const App = () => {
     ReactGA.initialize(config.GOOGLE_ANALYTICS_ID);
     ReactGA_4.initialize(config.GOOGLE_ANALYTICS_GA4_ID);
     return history.listen((loc) => {
-      const page = loc.pathname + loc.search;
-      ReactGA_4.send({
-        hitType: "pageview",
-        page,
-      });
-
-      ReactGA.set({ page });
-      ReactGA.pageview(page);
+      setTimeout(() => {
```

Comment:
Maybe add a comment here describing why this timeout is needed? It'll be helpful in case if anyone besides us two is reading this code.

Reply:
Good call, thanks!